core/frontend/src/render/FrameStats.ts

```diff
@@ -139,4 +147,13 @@ export class FrameStatsCollector {
     if (this._shouldRecordFrame)
       this._end(entry);
   }
+
+  private static async _postTelemetry(frameStats: Readonly<FrameStats>): Promise<void> {
+    if (!IModelApp.authorizationClient || !IModelApp.authorizationClient.hasSignedIn)
+      return;
+
+    const requestContext = await AuthorizedFrontendRequestContext.create();

```

Comment:
@evanpreslar-bentley @calebmshafer 
This context is being created every frame.
Is this necessary?  We do want iModelId to be included.  Does this mean we should create this every frame?
How can we ensure that iModelId is included?

Reply:
it is absolutely insane to send statistics for every frame for every user. That's a huge amount of nearly useless information. Who says we need to to do this?

Reply:
Sending every frame is a bit concerning. We could average the values and send infrequently.
However, for now, this draft PR is on hold until we hear back from Joe G on whether a different initiative to track performance will be suitable.
fyi @mgooding